.changes/windows-modify-cmd-string-behaviour.md

```diff
@@ -0,0 +1,9 @@
+ ---
+"cli.rs": patch
+---
+
+Currently `beforeDevCommand` and `beforeBuildCommand` have an issue where they can't deal with quotes properly on windows. Simple example is setting `beforeBuildCommand` to `ECHO %CD% && mkdir \"src-tauri\\websrc\"` which will first echo out the expected project repository, then wrongly create the directory `C:\src-tauri\websrc`.
+
+We just wish to run the command provided in the configuration without extra parsing. Without `/S` the user must escape certain characters for no benefit as we just wish to run whatever is entered in the configuration. With `/S` the user provided command will just execute as-is.
+
+See `CMD /?` for official documentation.
```

Comment:
I think a really short sentence, like "beforeDevCommand and x are now run with the /S option" or something like that (i'm not good with words haha). Having the explanations in the PR itself is enough and our changelog is already _huuuge_

Reply:
@FabianLars I have shortened it significantly.

Reply:
Wops, typo, 2s.